tox.ini

```diff
@@ -11,7 +11,8 @@ basepython =
     py3.6: python3.6
 deps =
     django1.11: Django>=1.11,<1.12
-    django2.0: Django>=2.0,<2.1
+    django2.0: Django>=2.0,<2.2
```

Comment:
Wrong upper bound ;)

Reply:
Dang. Yeah I started with just changing 2.0 to 2.1...